src/packages/sdk/package.json

```diff
@@ -76,7 +76,7 @@
     "string-width": "^4.2.0",
     "strip-ansi": "6.0.0",
     "strip-indent": "3.0.0",
-    "tar": "^6.0.1",
+    "tar": "^6.1.12",
```

Comment:
6.1.2 is what you want I think: https://www.npmjs.com/package/tar

Reply:
```suggestion
    "tar": "^6.1.2",
```

Reply:
lol good catch